docs/topics/multiplatform-mobile/multiplatform-mobile-create-first-app.md

```diff
@@ -1,66 +1,75 @@
-[//]: # (title: Create your first cross-platform mobile app – tutorial)
+[//]: # (title: Create your first cross-platform app)
 
-Here you will learn how to create and run your first Kotlin Multiplatform Mobile application.
+<microformat>
+    <p>This is a part of the <strong>Getting started with Kotlin Multiplatform Mobile</strong> tutorial. Before proceeding, make sure you've completed the previous step:</p>
+    <p><img src="icon-1-done.svg" width="20" alt="First step"/> <a href="multiplatform-mobile-setup.md">Set up environment</a><br/><img src="icon-2.svg" width="20" alt="Second step"/> <strong>Create your first cross-platform app</strong><br/><img src="icon-3-todo.svg" width="20" alt="Third step"/> Add dependencies<br/><img src="icon-4-todo.svg" width="20" alt="Fourth step"/> Upgrade your app<br/><img src="icon-5-todo.svg" width="20" alt="Fifth step"/> Wrap up your project</p>
+</microformat>
 
-1. [Set up an environment](multiplatform-mobile-setup.md) for cross-platform mobile development by installing the necessary tools on a suitable operating system.  
+Here you will learn how to create and run your first Kotlin Multiplatform Mobile application using Android Studio.
 
-    >You will need a Mac with macOS to complete certain steps in this tutorial, which include writing iOS-specific code and running an iOS application.  
-    >These steps cannot be performed on other operating systems, such as Microsoft Windows. This is due to an Apple requirement.
-    >
-    {type="note"}
+## Create the project from template
 
-You can also watch a video of this tutorial created by Ekaterina Petrova, Kotlin Developer Advocate.
-
-<video href="GcqFhoUuNNI" title="Kotlin Multiplatform Multiverse, Episode 2: Your First Kotlin Multiplatform Mobile App Tutorial"/>
+> You can also watch a [video of this tutorial](https://www.youtube.com/watch?v=GcqFhoUuNNI) created by Ekaterina Petrova, Kotlin Product Marketing Manager.
+>
+{type="tip"}
 
-2. In Android Studio, select **File** | **New** | **New Project**.
-3. Select **Kotlin Multiplatform App** in the list of project templates, and click **Next**.  
+1. In Android Studio, select **File | New | New Project**.
+2. Select **Kotlin Multiplatform App** in the list of project templates, and click **Next**.  
 
     ![Mobile Multiplatform project template](multiplatform-mobile-project-wizard-1.png)
     
-4. Specify a name for your first application, and click **Next**.  
+3. Specify a name for your first application, and click **Next**.  
```

Comment:
Could we update a screenshot and type a name without namespaces? For example, KotlinMultiplatformSandbox. I believe it's a good style to avoid namespaces although they're allowed